views/projects.tpl

```diff
@@ -94,7 +95,7 @@
 
             ["work", "categories", "applications", "lab"].forEach(id => {
                 let select = $(`#${id}`)[0];
-		  select.selectedIndex = 0;
+		        select.selectedIndex = 0;
```

Comment:
please undo automated indent

Reply:
tab / space mixup, I'm surprised that `prettier` didn't catch this.